app/Component/Compile.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import jsx from 'jsx-transform';
+
+class Compile extends React.Component {
+    evalInContext(template, context) {
+        /*jshint evil:true*/
+        with (context) {
+            return eval(jsx.fromString(template, context));
```

Comment:
Eval is evil, but it's the same mechanism in [ng-admin](https://github.com/marmelab/ng-admin/blob/0945d2ccb4cb392e3ec7e1eee26a7c53fbbeac30/src/javascripts/ng-admin/Crud/misc/Compile.js#L16)